.github/workflows/ci.yml

```diff
@@ -335,12 +335,12 @@ jobs:
                   docker/sdk testing codecept run -c codeception.functional.yml
 
 
-    php-80-mariadb-glue-alpine:
-        name: "PHP 8.0 / MariaDB / Glue / Alpine"
+    php-82-mariadb-glue-alpine:
```

Comment:
Everything that was 8.0 should be 8.1, everything that was 8.1 should be 8.2 please check all other places